internal/app/machined/pkg/controllers/kubespan/manager.go

```diff
@@ -474,6 +481,7 @@ func (ctrl *ManagerController) Run(ctx context.Context, r controller.Runtime, lo
 				spec.Kind = "wireguard"
 				spec.Up = true
 				spec.Logical = true
+				spec.MTU = mtu
```

Comment:
1420 should be the default for Wireguard link, but we can set it explicitly

Reply:
I think, default wireguard mtu size can be changed.
I want to make sure it will be pre-defend size... because the same size used in route table.

Reply:
yep, it's correct size, it's `1500 - (size of IPv6 + UDP headers)`